tests/spacetime/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+LINK_LIBRARIES(
+	spacetime-types
+	spacetime
+)
+
+ADD_CXXTEST(SpaceServerUTest)
```

Comment:
why create a new directory? why not just put it into the spatial directory?  There already is a spaceTest there, and a time test.